armsrc/appmain.c

```diff
@@ -2394,12 +2398,8 @@ static void PacketReceived(PacketCommandNG *packet) {
                     Dbprintf("transfer to client failed ::  | bytes between %d - %d (%d) | result: %d", i, i + len, len, result);
             }
             // Trigger a finish downloading signal with an ACK frame
-            // iceman,  when did sending samplingconfig array got attached here?!?
             // arg0 = status of download transfer
-            // arg1 = RFU
-            // arg2 = tracelen?
-            // asbytes = samplingconfig array
-            reply_mix(CMD_ACK, 1, 0, BigBuf_get_traceLen(), getSamplingConfig(), sizeof(sample_config));
```

Comment:
yeah,   this one clearly has some kind of meaning.   I am not sure where and when the attached sampleconfig is used on the client side.   Did you find where?

Reply:
I have only found one instance of CMD_DOWNLOAD_BIGBUF being used in the code
https://github.com/RfidResearchGroup/proxmark3/blob/29709a47b421e94d24816709c283ae158b9ad966/client/src/comms.c#L1144-L1147
all arguments are ignored when CMD_ACK is received in dl_it.
https://github.com/RfidResearchGroup/proxmark3/blob/29709a47b421e94d24816709c283ae158b9ad966/client/src/comms.c#L1192-L1193